src/appleseed/renderer/api/camera.h

```diff
@@ -36,6 +36,7 @@
 #include "renderer/modeling/camera/cameratraits.h"
 #include "renderer/modeling/camera/icamerafactory.h"
 #include "renderer/modeling/camera/orthographiccamera.h"
+#include "renderer/modeling/camera/perspectivecamera.h"
```

Comment:
Not sure it's relevant to include that header file here, it's not really part of the API since `PerspectiveCamera` is a private implementation detail, no?